scipy/stats/_distn_infrastructure.py

```diff
@@ -2339,14 +2339,22 @@ def integ(x):
     def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
                conditional=False, **kwds):
         """Calculate expected value of a function with respect to the
-        distribution.
+        distribution by numerical integration.
 
         The expected value of a function ``f(x)`` with respect to a
         distribution ``dist`` is defined as::
 
-                    ubound
-            E[x] = Integral(f(x) * dist.pdf(x))
-                    lbound
+                    ub
+            E[f(x)] = Integral(f(x) * dist.pdf(x)),
```

Comment:
One minor thing (not critical at all): i would write X or Z instead of x or change integration w.r.t. z

Reply:
@chrisb83, I see the distinction assumes the usual probability/statistics notational distinction of capital X for random variables and lower case x for the value the random variable takes on, i.e. Pr(X <x). That is a minor change, I'll include with the lambda function suggestion change from @josef-pkt. These will go into the next push, should only take a couple minutes. Any other objections from either of you? :)

Reply:
distinguishing between random variable X and a realised value, or integration variable x, sounds "too expert" to me.

(i.e. I can imagine getting issues about the "misspelled" capital X)

(i.e. when the target is not a math/econometrics journal:
"To simplify notation x stands for both random variables and a realization of it. The meaning should be obvious from the context."
)

Reply:
ok. Left the text clarifying the impact of lb, ub arguments and the conditional=True/False and switched the `X` back to lower case`x`. Changing the notation is indeed expert, and would require a cascade of changes to be consistent across the documentation. Anything else?

Reply:
ah, I undertsand @chrisb83 's point now. the r.h.s. (right hand side) of the integral is integrating over `x` and so the value on the l.h.s. of the integral should not be the same variable of integration. 
I will change the l.h.s. inside the expectation to read `E(f(z))`, eagle eyes! 👍

Reply:
hmm, maybe not, if we consider `E(x)` as a operator, putting the same `x` on l.h.s. and r.h.s. should be fine and unambiguous. I will leave as is (x on both l.h.s. and r.h.s.) unless @chrisb83 has an objection and clarifies.